Sources/Sentry/Public/SentryAppStartMeasurement.h

```diff
@@ -64,6 +64,11 @@ SENTRY_NO_INIT
  */
 @property (readonly, nonatomic, strong) NSDate *didFinishLaunchingTimestamp;
 
+@property (nonatomic, assign) uint64_t appStartSystemTime;
+@property (nonatomic, assign) uint64_t runtimeInitSystemTime;
+@property (nonatomic, assign) uint64_t moduleInitializationSystemTime;
+@property (nonatomic, assign) uint64_t didFinishLaunchingSystemTime;
```

Comment:
`h`: Do we need that information on the server? If we only need it for profiling on the client side to be able to add profiles to concurrent transactions, I think we must not make it public.